requirements/base.txt

```diff
@@ -16,4 +16,4 @@ httplib2
 sword2
 #python-keystoneclient==1.3.0
 #python-swiftclient==2.4.0
-
+django-celery-with-redis>=3.0
```

Comment:
I think it makes more sense to install celery and redis separately, to be more explicit about what versions of things we're installing ind if we change the backend.  Looks like it's `django-celery>=3.0,<4.0` and `redis>=2.4.4`